Makefile

```diff
@@ -27,6 +27,13 @@ sdk:
 clean-sdk:
 	cd gen/third_party/rplidar_sdk-release-${VERSION}/sdk && $(MAKE) clean_sdk
 
+install-swig:
+ifeq ("Darwin", "$(shell uname -s)")
```

Comment:
Check for brew existing instead of mac-specific test. Non-debian linux can still be supported that way.